jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/workitem/ServiceTaskViewImpl.java

```diff
@@ -83,9 +89,23 @@
     @AutoBound
     private DataBinder<ServiceTaskSummary> serviceTasks;
 
+    @Inject
+    @DataField("remove-service-task")
+    @SuppressWarnings("PMD.UnusedPrivateField")
+    private HTMLAnchorElement removeServiceTaskLink;
+
+    @Inject
+    @Named("span")
+    @DataField("remove-service-task-value")
+    private elemental2.dom.HTMLElement removeServiceTaskValue;
+
+    @Inject
+    private Event<RemoveServiceTaskEvent> removeServiceTaskEvent;
```

Comment:
This type of logic should be in the presenter. Please use a Command as callback, set from the presenter in here.
```suggestion
    private Command callback;
```